tests/test_annotated_types.py

```diff
@@ -290,3 +290,54 @@ class Model(BaseModel):
 
     with pytest.raises(ValidationError):
         Model.parse_obj({'t': {'v': -1}})
+
+
+def test_typeddict_required_not_required():
```

Comment:
can we separate this into multiple tests.

Reply:
fixed, thanks